index.html

```diff
@@ -40,6 +46,26 @@
             openmct.install(openmct.plugins.Espresso());
             openmct.install(openmct.plugins.Generator());
             openmct.install(openmct.plugins.UTCTimeSystem());
+
+            /*
+                Will be installed by default...somehow
+             */
+            //openmct.install(ConductorService());
+
+            var ONE_MINUTE = 60 * 1000;
+            var ONE_YEAR = 365 * 24 * 60 * 60 * 1000;
+
+            // Will also provide a default configuration based on enabled time
+            // systems and tick sources.
+            openmct.install(openmct.plugins.Conductor({
```

Comment:
After discussion, the public API could allow configuration to be set directly, which the Conductor plugin could use to configure the conductor from provided configuration.